src/main_executor.js

```diff
@@ -41,6 +41,7 @@ const argv = require("yargs")
     .alias("e", "execute")
     .alias("v", "verbose")
     .alias("T", "tracer")
+    .alias("nc", "nocounters")
```

Comment:
IMO would have been better to use one letter, no two letters.

Reply:
agree! I have chamge it xD